docs/pages/reference/access-controls/authentication.mdx

```diff
@@ -267,3 +267,50 @@ See [GitHub OAuth 2.0](../../admin-guides/access-controls/sso/github-sso.mdx) fo
 
 </TabItem>
 </Tabs>
+
+## Require Displaying Message of the Day (MOTD)
+
+Teleport can display a custom MOTD for users prior to authenticating
+in the Teleport Web UI and CLI. 
+
+<Tabs>
+<TabItem scope={["enterprise", "oss"]} label="Self-Hosted">
+
+Add the following to your Teleport configuration file, which is stored in
+`/etc/teleport.yaml` by default.
+
+  ```yaml
+  auth_service:
+  message_of_the_day: "\n Welcome to the Example Teleport Cluster\n  All activity is monitored and should follow organization policies\n"
+  ```
+
+  Restart the Teleport Auth services to apply this change.
```

Comment:
Why are we capitalizing the name of the Teleport Auth Service like this?

Reply:
@ptgott please see updates.